src/main/sensors/acceleration.c

```diff
@@ -348,9 +352,11 @@ bool accInit(uint32_t gyroSamplingInverval)
             biquadFilterInitLPF(&accFilter[axis], accLpfCutHz, acc.accSamplingInterval);
         }
     }
+#if 0
```

Comment:
This should be removed - we should probably excise all references to `acc_align`.